src/lib/rpc.ts

```diff
@@ -15,7 +21,10 @@ export const networkConfigUrl = (agoricNetName: string) => {
     return `https://${agoricNetName}.agoric.net/network-config`;
   }
 };
-export const rpcUrl = agoricNetSubdomain =>
```

Comment:
Odd that the diff thinks this was removed. Consider adjusting line ordering so the line history is correct

Reply:
Attempted, will see if that worked